src/main/java/com/github/manolo8/darkbot/gui/titlebar/MainTitleBar.java

```diff
@@ -31,21 +41,100 @@ public MainTitleBar(Main main, MainGui frame) {
     }
 
     public void setInfo(String info) {
-        this.info.label.setText(info);
+        this.info.title.setText(info);
     }
 
-    private static class Info extends Box.Filler {
+    private class Info extends Box.Filler {
 
-        private final JLabel label = new JLabel();
+        private final JLabel title = new JLabel();
 
         Info() {
             super(new Dimension(20, 0), new Dimension(120, 0), new Dimension(Short.MAX_VALUE, Short.MAX_VALUE));
             setLayout(new BorderLayout());
             setBorder(BorderFactory.createEmptyBorder(0, 5, 0, 5));
-            add(this.label, BorderLayout.CENTER);
-            label.setHorizontalAlignment(SwingConstants.CENTER);
+
+            StatsManager statsManager = main.statsManager;
+
+            add(title, BorderLayout.CENTER);
+            add(createLeftPanel(statsManager), BorderLayout.WEST);
+            if (Main.API.hasCapability(Capability.HANDLER_CPU_USAGE) && Main.API.hasCapability(Capability.HANDLER_RAM_USAGE)) {
+                add(createRightPanel(statsManager), BorderLayout.EAST);
+            }
+            title.setHorizontalAlignment(SwingConstants.CENTER);
         }
 
-    }
+        private JComponent createLeftPanel(StatsManager statsManager) {
+            JPanel panel = new JPanel(new GridLayout(2, 1));
+
+            JLabel tick = createLabel("tick_time", false, null,
+                    () -> "Tick time\n\n" + statsManager.getTickStats());
+            JLabel ping = createLabel("ping", false,
+                    color -> {
+                        float hue = 1 - Math.min(500, Math.max(1, main.statsManager.getPing())) / 500f;
```

Comment:
i think the scale here may not be all that great; 500 ping is real bad. given the 500 is purely for color purposes, i think past 300ms getting the worst possible red is fine, with the green-to-red going from 1ms to 300ms.